src/files.js

```diff
@@ -159,6 +159,7 @@ FOAM_FILES([
   { name: "foam/json2/Outputter" },
   { name: "foam/json2/Serializer" },
   { name: "foam/json2/Deserializer" },
+  { name: "foam/java/Util" },
```

Comment:
change to:
{ name: "foam/java/Util", flags: ['java'] },

So that it is only included when building Java, not for the web.

Reply:
OK